res/style.css

```diff
@@ -17,6 +17,25 @@ body, #root, .profileViewer {
   max-height: 100%;
 }
 
+body {
+  /* Stolen from the light theme at devtools/client/themes/variables.css */
+  --theme-highlight-green: #2cbb0f;
+  --theme-highlight-blue: #0088cc;
+  --theme-highlight-bluegrey: #0072ab;
+  --theme-highlight-purple: #5b5fff;
+  --theme-highlight-lightorange: #d97e00;
+  --theme-highlight-orange: #f13c00;
+  --theme-highlight-red: #ed2655;
+  --theme-highlight-pink: #b82ee5;
+  --theme-highlight-gray: #b4babf;/* except for this one, I made this one darker */
+}
+
+#root {
+  /* Create a stacking context so that tooltips can be placed over the root element. */
+  position: relative;
+  z-index: 0;
+}
```

Comment:
I'm not convinced `fixed` would be a better implementation for this behavior. I like the explicitness of creating a stacking context and controlling the structure of the parent containers. I'm a little scared of the implicit stacking behavior of fixed position things, mainly because I can't find great documentation on it, especially since the behavior changed ~2012 from looking into it.